java-client/src/main/java/org/scassandra/http/client/PrimingRequest.java

```diff
@@ -281,6 +286,53 @@ public Result getResult() {
         public long getFixedDelay() {
             return fixedDelay;
         }
+
+        public static class ThenBuilder {
```

Comment:
I'll add the then builder interface to 
Priming request and deprecate the old one so new features only need to be added to the then builder 


> On 18 Aug 2015, at 23:52, Andrew Tolbert <notifications@github.com> wrote:
> 
> In java-client/src/main/java/org/scassandra/http/client/PrimingRequest.java:
> 
> > @@ -281,6 +286,53 @@ public Result getResult() {
> >          public long getFixedDelay() {
> >              return fixedDelay;
> >          }
> > +
> > +        public static class ThenBuilder {
> It is weird that BatchPrimingRequest uses this but PrimingRequest doesn't, but understand it for API compatibility and I like the direction towards ThenBuilder.
> 
> —
> Reply to this email directly or view it on GitHub.
>